package.json

```
@@ -26,10 +26,14 @@
     "prepack": "oclif manifest && oclif readme",
     "pretest": "npm run lint && npm run typecheck",
     "readme": "npm run ci:fix && npm run build && npm exec oclif readme",
+    "release": "chmod +x scripts/release.sh && ./scripts/release.sh",
```

```suggestion
    "release": "chmod +x scripts/release.sh && ./scripts/release.sh",
```
Why chmod? if you chmod +x locally and push it, it will keep the right permission

---

Good point! Done [4809f9f](https://github.com/herodevs/cli/pull/187/commits/4809f9fccdec8fd8d01f7b00bd3724dc00a3fbf7)